test/trivial/diten/testIntExp4.good

```diff
@@ -1 +1 @@
-error: 0 cannot be raised to a negative power
```

Comment:
This changed because now USR_FATAL will use the guessed line number (and not warn about it, except in --developer mode). I found the line number to be accurate for this case...